exporter/coralogixexporter/logs_client.go

```diff
@@ -56,8 +56,16 @@ type logsExporter struct {
 }
 
 func (e *logsExporter) start(ctx context.Context, host component.Host) (err error) {
-	if e.clientConn, err = e.config.Logs.ToClientConn(ctx, host, e.settings, grpc.WithUserAgent(e.userAgent)); err != nil {
-		return err
+	switch {
+	case !isEmpty(e.config.Logs.Endpoint):
```

Comment:
Do you want the LogsEndpoint configuration to take precedence over Domain?  The config allows setting traces/metrics/logs endpoints and domain endpoint at the moment.